src/components/ThirdPartyCollectionDetailPage/CollectionItem/CollectionItem.tsx

```diff
@@ -53,7 +53,7 @@ export default class CollectionItem extends React.PureComponent<Props> {
       <Grid className={`CollectionItem ${styles.grid}`} columns="equal">
         <Grid.Row className={styles.row}>
           <Grid.Column className={`${styles.column} ${styles.avatarColumn}`} width={5}>
-            <Checkbox checked={selected} onClick={this.handleCheckboxChange} />
+            <Checkbox checked={selected || item.isPublished} disabled={item.isPublished} onClick={this.handleCheckboxChange} />
```

Comment:
This is true, I'll delete the check, we might want to revise adding some kind of sort for published and unpublished items. Right now, it'll apparent because of the ItemStatus component